components/Editor/services/editor.service.ts

```diff
@@ -19,5 +19,6 @@ export const createEditor = async ({
     placeholder,
     data: content,
     tools,
+    autofocus: true,
```

Comment:
you should add a prop for Editor component and use it only where it is necessary